tips/TIP-0049/tip-0049.md

```diff
@@ -0,0 +1,567 @@
+            <td>uint8</td>
+            <td>Bits Count is the number of bits used to represent Mana.</td>
+          </tr>
+          <tr>
+            <td>Generation Rate</td>
+            <td>uint8</td>
+            <td>Generation Rate is the amount of potential Mana generated by 1 IOTA in 1 slot.</td>
+          </tr>
+          <tr>
+            <td>Generation Rate Exponent</td>
+            <td>uint8</td>
+            <td>Generation Rate Exponent is the scaling of Mana Generation Rate expressed as an exponent of 2.</td>
+          </tr>
+          <tr>
+            <td>Decay Factors</td>
+            <td>(uint16)Uint32Array</td>
+            <td>Decay Factors is the number of decay factors used to calculate the decay of Mana.</td>
+          </tr>
+          <tr>
+            <td>Decay Factors Exponent</td>
+            <td>uint8</td>
+            <td>Decay Factors Exponent is the scaling of ManaDecayFactors expressed as an exponent of 2.</td>
+          </tr>
+          <tr>
+            <td>Decay Factor Epochs Sum</td>
+            <td>uint32</td>
+            <td>Decay Factor Epochs Sum is an integer approximation of the sum of decay over epochs.</td>
+          </tr>
+          <tr>
+            <td>Decay Factor Epochs Sum Exponent</td>
+            <td>uint8</td>
+            <td>Decay Factor Epochs Sum Exponent is the scaling of Decay Factor Epochs Sum expressed as an exponent of 2.</td>
+          </tr>
+        </table>
+      </details>
+    </td>
+  </tr>
+  <tr>
+    <td>Token Supply</td>
+    <td>uint64</td>
+    <td>Token Supply defines the current token supply on the network.</td>
+  </tr>
+  <tr>
+    <td>Genesis Slot</td>
+    <td>uint32</td>
+    <td>Genesis Slot defines the slot of the genesis.</td>
+  </tr>
+  <tr>
+    <td>Genesis Unix Timestamp</td>
+    <td>uint64</td>
+    <td>Genesis Unix Timestamp defines the genesis timestamp at which the slots start to count</td>
+  </tr>
+  <tr>
+    <td>Slot Duration In Seconds</td>
+    <td>uint8</td>
+    <td>Slot Duration In Seconds defines the duration of each slot in seconds.</td>
+  </tr>
+  <tr>
+    <td>Slots Per Epoch Exponent</td>
+    <td>uint8</td>
+    <td>Slots Per Epoch Exponent is the number of slots in an epoch expressed as an exponent of 2. (2**SlotsPerEpochExponent) == slots in an epoch.</td>
+  </tr>
+  <tr>
+    <td>Staking Unbonding Period</td>
+    <td>uint32</td>
+    <td>Staking Unbonding Period defines the unbonding period in epochs before an account can stop staking.</td>
+  </tr>
+  <tr>
+    <td>Validation Blocks Per Slot</td>
+    <td>uint8</td>
+    <td>Validation Blocks Per Slot is the number of validation blocks that each validator should issue each slot.</td>
+  </tr>
+  <tr>
+    <td>Punishment Epochs</td>
+    <td>uint32</td>
+    <td>The number of epochs worth of Mana that a node is punished with for each additional validation block it issues.</td>
+  </tr>
+  <tr>
+    <td>Liveness Threshold Lower Bound In Seconds</td>
```

Comment:
I wouldn't put the `In Seconds` here, this is a bit inconsistent with the rest as this is not the JSON name.